sli/fdstreamtest.cc

```diff
@@ -67,7 +67,8 @@ main()
   }
```

Comment:
As @jougs pointed out in another PR, this and the other files with formatting fixes should not be part of a pull request fixing a non-formatting issue.